poetry.lock

```diff
@@ -1,16 +1,10 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
please update Poetry on your system. you're most likely running an older version with a bug that destroyed the lockfile. when you look through the changes you'll notice it removed all the wheels and their hashes.

Reply:
You're right, I was on 1.2.2, updated to 1.4.0 now.

I think it just moved them at the end of the file, but in any case, it defo shouldn't be downgraded. fixed